src/components/NewCommentForm.tsx

```diff
@@ -45,24 +210,32 @@ export const NewCommentForm: React.FC = () => {
             name="email"
             id="comment-author-email"
             placeholder="email@test.com"
-            className="input is-danger"
+            className={classNames('input', {
+              'is-danger': !!error.email.length || !isEmailValid,
+            })}
+            value={author.email}
+            onChange={handleAuthEmailChange}
           />
 
           <span className="icon is-small is-left">
             <i className="fas fa-envelope" />
           </span>
 
-          <span
-            className="icon is-small is-right has-text-danger"
-            data-cy="ErrorIcon"
-          >
-            <i className="fas fa-exclamation-triangle" />
-          </span>
+          {(!!error.email.length || !isEmailValid) && (
```

Comment:
```suggestion
          {(!!error.email.length) && (
```
looks like it is reason of failed test